python/ray/serve/deployment_state.py

```diff
@@ -743,7 +743,7 @@ def resource_requirements(self) -> Tuple[str, str]:
         available = {
             k: v for k, v in self._actor.available_resources.items() if k in required
         }
-        return str(required), str(available)
+        return json.dumps(required), json.dumps(available)
```

Comment:
please leave a comment here for why we use `json`

Reply:
Good idea, I added a comment.